controllers/fenceagentsremediation_controller.go

```diff
@@ -182,10 +182,15 @@ func buildFenceAgentParams(far *v1alpha1.FenceAgentsRemediation) ([]string, erro
 		if paramName != parameterActionName {
 			fenceAgentParams = appendParamToSlice(fenceAgentParams, paramName, paramVal)
 		} else if paramVal != parameterActionValue {
-			logger.Info("FAR doesn't support any other action than reboot", "action", paramVal)
+			// --action attribute was selected but it is differemt than reboot
+			err := errors.New("FAR doesn't support any other action than reboot")
+			logger.Error(err, "can't build CR with this action attribute", "action", paramVal)
+			return nil, err
```

Comment:
I think it's fine returning nil here since I don't see how this `error` will be resolved.

Reply:
So you are suggesting to not return error and avoid exponential back off errors by reporting an error only once from the above line (187)?

Reply:
Yep, IMO it's enough to just log it.

Reply:
Correction: I wasn't aware that we aren't in reconcile context, so returning the error here is find, but I wouldn't return it in the Reconcile method